### PR TITLE
Add a technote for UCSCXenaTools

### PR DESCRIPTION
**Pull request by ShixiangWang**
https://github.com/ropensci/software-review/issues/315#issuecomment-517783222

### CODE REVIEW

**Comment by ShixiangWang** [comment]
@stefaniebutland Please review this pull request.

Thanks,
Shixiang

File: content/technotes/2019-09-06-ucscxenatools-surv.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+Before actually pulling data, understand how **UCSCXenaTools** [^1] works (see Figure 1) will help user locate the most important function to use.
+
+Generally,
+
+- for operating datasets, we use functions whose names start with `Xena`
+- for operating subset of a dataset, we use functions whose names start with `fetch_`
+
+![](/img/blog-images/2019-09-06-ucscxenatools-surv/overview.png)*Figure 1. The UCSCXenaTools pipeline*
+
+To illustrate how to download data and combine with other packages for analysis, here we retrieve expression data of gene [*KRAS*](https://ghr.nlm.nih.gov/gene/KRAS) (which is a known driver in LUAD) and survival status from [TCGA Lung Adenocarcinoma (LUAD)](https://xenabrowser.net/datapages/?cohort=TCGA Lung Adenocarcinoma (LUAD)&removeHub=https%3A%2F%2Fxena.treehouse.gi.ucsc.edu%3A443) as example data to finish a survival analysis procedure, which is frequently shown in cancer researches.
```
[the remark this answers is not in the document]
Comment (ShixiangWang) [reply content/technotes/2019-09-06-ucscxenatools-surv.md]: Nice editing! But how can I use your version here?

**Comment by ShixiangWang** [comment]
@stefaniebutland Thanks for your comments and editing. I don't know how to modify the file in this pull request based on some of your comments.

**Comment by ShixiangWang** [comment]
@stefaniebutland Thanks, Your suggestion is what I need. I am not a native English speaker, so your suggestion improves the quality of the technote.

But I cannot apply the "suggested changes" according to the following message from https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request

> Other people can suggest specific changes to your pull request. You can apply these suggested changes directly in a pull request if you have write access to the repository. If the pull request was created from a fork and the author allowed edits from maintainers, you can also apply suggested changes if you have write access to the upstream repository

I will push commits according to your "suggested changes" and comments by hand as soon as possible and tag you again.

Best,
Shixiang

**Comment by stefaniebutland** [comment]
> I am not a native English speaker

Your writing was very clear!

Sorry I wasn't able to explain how to commit the others, but there aren't too many so manually won't take so long.

Cheers,
Stefanie

**Comment by ShixiangWang** [comment]
@stefaniebutland Please review the latest commit, thanks :).

**Comment by stefaniebutland** [comment]
@ShixiangWang Thank you for those changes. I'll make a few more edits directly on your pull request and will reply here for you to check

**Comment by stefaniebutland** [comment]
Hi @ShixiangWang. I made some more edits directly and listed them below. If you disagree with any, please let me know. You can see these in the post preview: https://deploy-preview-510--ropensci.netlify.com/technotes/2019/09/06/ucscxenatools-surv/

- added "the", "an"
- link to TCGA in first sentence [The Cancer Genome Atlas (TCGA)](https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga)
- changed "LUAD patients from the TCGA (The Cancer Genome Atlas) program" -> "LUAD patients from the TCGA" since abbreviation defined above
- in Acknowledgements, added links to reviewers' GitHub
- edited Figure 2 heading to "*Figure 2. Kaplan-Meier curve. Survival probability vs Time (days)*", for even more clarity than having "days" only in the text below plot
- no boldface on the package name UCSCXenaTools
- Is my edit appropriate here? "use Kaplan-Meier curve and log-rank test to observe the difference in different ofKRAS gene expression status" -> "use Kaplan-Meier curve and log-rank test to observe the effect of different KRAS gene expression status"

Question: This text on XenaShiny,
> XenaShiny, a Shiny project based on UCSCXenaTools, is under development by me and some friends. You can take a try if you have little programming experience.

could be
> XenaShiny, a Shiny project based on UCSCXenaTools, is under development by my friends and me.

Can you please make that change?

That should be the final edit needed before publication on Sep 6.

**Comment by ShixiangWang** [comment]
@stefaniebutland All excellent changes. Could you change them all directly? I am totally agree with your suggestions. Thank you for helping me:heartbeat:.

**Pull request merged**